### PR TITLE
Adding a section describing a wireless connection of PS 4 controller.

### PR DESCRIPTION
**Pull request by kaklik**

### CODE REVIEW

**Comment by jaxxzer** [comment]
This worked like a charm for me.

**Comment by jaxxzer** [comment]
I suggest you take out the jstest-gtk stuff, and just run qgc, and go to the joystick page to make sure it's working.

Also, you might add instructions to run ds4drv at startup: add the line `ds4drv --daemon` to `/etc/rc.local`

File: en/SetupView/Joystick.md [hunk 1]
```diff
@@ -43,6 +43,30 @@ The following joysticks/controllers have been shown to work with relatively rece
 
 These are both highly recommended. They work well "out of the box" and have many buttons that you can map to flight modes. 
 
+#### Sony PS4 - DualShock 4 Controller V2
+
+This type allows both USB and wireless connection (Bluetooth). Wired USB connection usually works out of the box. The wireless connection needs some setup. 
+
+##### Linux Ubuntu setup
```
Comment (hamishwillee) [inline en/SetupView/Joystick.md]: Thanks @kaklik !

@DonLakeFlyer Can we automate this bit of setup as part of the installation? I'd rather not include developer instructions in the user guide.

Comment (DonLakeFlyer) [reply en/SetupView/Joystick.md]: I don't know anything about DualShock wireless setup. I'd assume this refers to setup with respect to the OS which QGC would have nothing to do with.

**Comment by jaxxzer** [comment]
> I'd rather not include developer instructions in the user guide.

We could instead document some 'out of the box' compatibility matrix for controller + os combinations that work with no such setup. We could also limit support to the combinations listed here.

**Comment by hamishwillee** [comment]
@jaxxzer 

> This worked like a charm for me.

You mean following these instructions for using the controller wirelessly?

> I suggest you take out the jstest-gtk stuff, and just run qgc, and go to the joystick page to make sure it's working.

@kaklik 
- I think the suggestion here is that the jstest-gtk isn't needed - can you retest?
- also add instructions to run ds4drv at startup: add the line `ds4drv --daemon to /etc/rc.local`

>> I'd rather not include developer instructions in the user guide.

> We could instead document some 'out of the box' compatibility matrix for controller + os combinations that work with no such setup. We could also limit support to the combinations listed here.

In theory, but I think we're better off not trying to constrain people, since we don't own the evolution of the underlying joystick API. So I will live with my dislike. After all open issues are clarified I'll submit and subedit to my liking :-)

**Comment by kaklik** [comment]
I think that jstest-gtk is needed because by default the touchpad at PS4 controller is linked to mouse control. After installation of jstest-gtk package (and its dependencies) the touchpad is locked to joystick axis.

**Comment by hamishwillee** [comment]
OK. Thanks. Merging

**Comment by jaxxzer** [comment]
> In theory, but I think we're better off not trying to constrain people, since we don't own the evolution of the underlying joystick API. So I will live with my dislike. After all open issues are clarified I'll submit and subedit to my liking :-)

It looks like we actually have what I was indicating in [Supported Joysticks](https://docs.qgroundcontrol.com/en/SetupView/Joystick.html#supported-joysticks)

**Pull request merged**